src/app/urls.py

```diff
@@ -27,6 +27,7 @@
 urlpatterns = [
     path('', views.index, name='index'),
     path('validate/', views.validate, name='validate'),
+    path('checker/', views.ntia_check, name='checker'),
```

Comment:
should we keep the url as `ntia_checker/` ? Is NTIA easily understandable by the user? Or any other url as checker and validate sounds similar. Users should be able to distinguish b/w them.

Reply:
Ok I will change it.